tahoma_api/client.py

```diff
@@ -141,6 +141,16 @@ async def get_current_executions(self) -> List[Any]:
 
         return response
 
+    async def get_scenarios(self) -> List[Scenario]:
+        """ List the scenarios """
+        response = await self.__get("actionGroups")
+        return [Scenario(**scenario) for scenario in response]
+
+    async def launch_scenario(self, oid: str) -> str:
```

Comment:
Can we change launch? I would like to align it to command as well in the end.

For example `execute_command` and `execute_scenario`?

Reply:
Sure, I will change this.

Reply:
Done.